app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -1680,6 +1684,38 @@ class PluginApi {
   addTopicsSectionLink(arg) {
     addSectionLink(arg);
   }
+
+  /**
+   * Decorate inline bbcode in to-markdown function.
+   *
+   * ```
+   * api.beforeToMarkdownTagDecorate(function (text) {
+   *   if (this.element.attributes.class === "spoiled") {
+   *     this.prefix = "[spoiler]";
+   *     this.suffix = "[/spoiler]";
+   *   }
+   * });
+   * ```
+   */
+  beforeToMarkdownTagDecorate(callback) {
```

Comment:
What you call tag is what we call inline at other places right?

Reply:
In most cases - I think so? But since `Tag` is the base class for all tags, its decorate method is called for e.g. `blockquote`, `td`, `th` - so also elements that are not strictly inline.